semantic_conventions.yaml

```diff
@@ -0,0 +1,32 @@
+# This file describes standards and conventions for Span tags and logs.
```

Comment:
I can't remember where (it wasn't on this PR), but there were some people who wanted it to continue to exist within the past week.